resources/tap.brs

```diff
@@ -127,3 +128,7 @@ function __tap_getIndent()
     end for
     return indent
 end function
+
+function __tap_bail()
+    print "Bail out!"
```

Comment:
TIL!  This makes a ton of sense.

How would you feel about supporting a `msg ="" as string` optional argument so we can do something like `tap.bail("Error occurred in " + path)`?

http://testanything.org/tap-version-13-specification.html#bail-out

Reply:
I like that a lot! I'll add it 👍

Reply:
Turns out `tap-mocha-reporter` doesn't seem to respect the extra message. I'll still add it, in case we want to figure it out at some point in the future.